test/global_state_test.ts

```diff
@@ -303,18 +446,27 @@ async function exportCircuitAndTestData(blocks, component) {
   await snarkit.utils.writeCircuitIntoDir(circuitDir, component);
   for (let i = 0; i < blocks.length; i++) {
     const block = blocks[i];
-    await snarkit.utils.writeInputOutputIntoDir(path.join(dataDir, printf('%04d', i)), block, {});
+    await snarkit.utils.writeInputOutputIntoDir(path.join(dataDir, printf('b%04d', i)), block, {});
```

Comment:
what is `b%04d`?

Reply:
It was for debug purpose (I put the data generated from rollup-manager's block test case and from global_state_test.ts together so it would be easy to compare the input) and I have just forgotten to resume the change ...

This change can be discarded.